apps/docs/content/blog/v2.4.0.mdx

```diff
@@ -266,8 +266,20 @@ you need to update the import as follows:
 + import {cn} from "@nextui-org/theme"
 ```
 
-<Spacer y={4} />
+### Validation Behavior
+
+Since v2.4.0, we've changed the default validation behavior to `aria` wihch means no native validation is applied. If you wish to use the native validation, you can set `validationBehavior` to `native` to the input components or set it to the Provider as stated above.
```

Comment:
```suggestion
Since v2.4.0, we've changed the default validation behavior to `aria` which means no native validation is applied. If you wish to use the native validation, you can set `validationBehavior` to `native` to the input components or set it to the Provider as stated above.
```

Reply:
Good catch.